Add O1NumHess as dependency

stgeo opened the pull request:
Added the O1NumHess repository as a dependency in setup.py, so that it is installed automatically.

stgeo committed:
add O1NumHess as dependency

ilcpm commented:
thanks, sorry for the late action

ilcpm commented:
I just added link for O1NumHess in readme, the code you added in setup.py may cause problem in some cases (such as no network), so I deleted it, sorry 😂

stgeo commented:
> I just added link for O1NumHess in readme, the code you added in setup.py may cause problem in some cases (such as no network), so I deleted it, sorry 😂

Ideally, the packages are made available on pypi so that a python dependency manager (pip, pdm, poetry, uv) can install them automatically. Since they aren't (or at least weren't at the time), using the git repo link was a possible workaround. This allows the `pip install .` command in the readme to actually work out of the box. IMO, a link in the readme is not a user-friendly way to provide the dependencies of your package, but as you prefer - it's your code :wink:
